v4/export/config.go

```diff
@@ -154,6 +154,7 @@ const (
 	DefaultTiDBMemQuotaQuery = 32 * (1 << 30)
 	DefaultStatementSize     = 1000000
 	TiDBMemQuotaQueryName    = "tidb_mem_quota_query"
+	DefaultTableFilter       = "!/^(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA)$/.*"
```

Comment:
```suggestion
	DefaultTableFilter       = "!/^(mysql|sys|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA)$/.*"
```

i see no reason excluding the `test` schema. but [`sys`](https://dev.mysql.com/doc/mysql-monitor/8.0/en/mem-graphs-sys-schema-install-ref.html) *is* a system schema.

(btw why not `[]string{"*.*", "!mysql.*", "!sys.*", "!INFORMATION_SCHEMA.*", "!PERFORMANCE_SCHEMA.*", "!METRICS_SCHEMA.*", "!INSPECTION_SCHEMA.*"}`)

Reply:
Yes. I made a mistake. It should be `sys` here.

Because I think that's easier for users to paste if they want to exclude some tables and system tables.
If users want to exclude some databases they can type fewer lines if they have seen the help information. Or, they need to type `--f '*.*' -f '!mysql.*' ... -f '${the tables they want to exclude}'`